README.md

```diff
@@ -106,20 +106,35 @@ This repository is a [Node.js](https://nodejs.org) web application that specifie
 - Versions are locked and managed using [npm](https://www.npmjs.com/)
 - Scales across processor cores in larger dynos via [Node cluster API](https://nodejs.org/dist/latest-v10.x/docs/api/cluster.html)
 
+
+## Using a new theme
+
+Due to the Ephimeral file system of Heroku, currently the admin panel doesn't support uploading the themes directly. If uploaded, the theme would go missing after some time. The only way to use custom theme currently is to commit your themes in the code files and then re publish the application.
+
+**To use a new theme follow these steps**
+
+1. Fork and clone this repository
+2. Extract and copy your theme folder in content/themes. If your theme is `Editorial` it should look like `content/themes/Editorial/...`
+3. Push the change
+4. Deploy to Heroku by clicking the button below
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```

Comment:
I wish you would at least look through the closed issues list before making changes on your own. See #32 for why the querystring is necessary. Revert this change.

Reply:
Got it, I'll change that